group_vars/all.yml

```diff
@@ -134,3 +134,6 @@ sync_db_driver: org.postgresql.Driver
 sync_db_name: "alfresco-sync"
 sync_db_username: "alfresco"
 sync_db_password: "alfresco"
+
+enable_ssl: false
```

Comment:
Just a naming comment, could we please stick to the convention we started with "use_custom_keystores" i.e. prefixing any feature flags with "use_" e.g. "use_ssl"